app/app/wyoming.ts

```diff
@@ -1,7 +1,12 @@
 // exposes the microphone as a wyoming satellite
 import TcpSocket from 'react-native-tcp-socket';
 import {HMLogger} from './logger';
-import {APP_VERSION, AUDIO_INFO, MIC_GAIN, WYOMING_PORT} from './constants';
+import {
+  APP_VERSION,
+  AUDIO_INFO,
+  STORAGE_MIC_GAIN,
```

Comment:
not using this anymore; we can delete it

Reply:
removed